fenix/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -34,17 +34,11 @@
         app:popUpToInclusive="false" />
 
     <action
-        android:id="@+id/action_global_home_juno_onboarding"
+        android:id="@+id/action_global_juno_onboarding"
```

Comment:
nit: since we're here, do you think it's worth reducing the juno name references here or later as part of BZ-1826883?

```suggestion
        android:id="@+id/action_global_new_user_onboarding"
```

Reply:
Yes definitely, later as part of the linked bug, will update all the references. Since there are still some updates coming in the onboarding, will be safe to do that as the last thing to wrap it up.